<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>details</class>
 <widget class="QDialog" name="details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="screenshotButton">
   <property name="geometry">
    <rect>
     <x>652</x>
     <y>667</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ScreenShot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_print">
   <property name="geometry">
    <rect>
     <x>652</x>
     <y>627</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PRINT</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetConteneur" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>781</width>
     <height>621</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #000000;                /* Couleur noire */
    font-weight: bold;             /* Texte en gras (facultatif) */
}
</string>
    </property>
    <property name="text">
     <string>Employer ID :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Statue">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #000000;                /* Couleur noire */
    font-weight: bold;             /* Texte en gras (facultatif) */
}
</string>
    </property>
    <property name="text">
     <string>Ticket ID :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #000000;                /* Couleur noire */
    font-weight: bold;             /* Texte en gras (facultatif) */
}
</string>
    </property>
    <property name="text">
     <string>Title :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ID_Dep">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ID_Emp">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Desc">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #000000;                /* Couleur noire */
    font-weight: bold;             /* Texte en gras (facultatif) */
}
</string>
    </property>
    <property name="text">
     <string>Status :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>510</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #000000;                /* Couleur noire */
    font-weight: bold;             /* Texte en gras (facultatif) */
}
</string>
    </property>
    <property name="text">
     <string>Description :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ID_Ticket">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Titre">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #000000;                /* Couleur noire */
    font-weight: bold;             /* Texte en gras (facultatif) */
}
</string>
    </property>
    <property name="text">
     <string>Departement ID :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #000000;                /* Couleur noire */
    font-weight: bold;             /* Texte en gras (facultatif) */
}
</string>
    </property>
    <property name="text">
     <string>Date :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Type">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #000000;                /* Couleur noire */
    font-weight: bold;             /* Texte en gras (facultatif) */
}
</string>
    </property>
    <property name="text">
     <string>Type :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Date">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
